guides/doc-Configuring_User_Authentication/docinfo.xml

```diff
@@ -0,0 +1,15 @@
+<title>{ConfiguringUserAuthenticationDocTitle}</title>
+<productname>Red Hat Satellite</productname>
```

Comment:
```suggestion
<productname>{ProjectName}</productname>
```

https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L110

Unless XML does not like the nbsp :shrug:

Reply:
A good idea, but all other docinfo.xml's use the product name spelled out. I'd rather not diverge from the other guides.